spec/classes/storage_spec.rb

```diff
@@ -21,30 +21,14 @@
       case facts[:os]['family']
       when 'Debian'
         it { is_expected.to contain_package('bacula-sd') }
-
-        case facts[:os]['release']['major']
-        when '7', '8'
-          it { is_expected.to contain_package('bacula-sd-pgsql') }
-        end
       when 'RedHat'
-        case facts[:os]['release']['major']
-        when '6'
-          it do
-            is_expected.to contain_package('bacula-storage-common').with(
-              'ensure' => 'installed'
-            )
-          end
-
-          it { is_expected.not_to contain_package('bacula-storage') }
-        else
-          it do
-            is_expected.to contain_package('bacula-storage').with(
-              'ensure' => 'installed'
-            )
-          end
-
-          it { is_expected.not_to contain_package('bacula-storage-common') }
+        it do
+          is_expected.to contain_package('bacula-storage').with(
+            'ensure' => 'installed'
+          )
         end
+
+        it { is_expected.not_to contain_package('bacula-storage-common') }
```

Comment:
Since this package is never mentioned in the source anymore I'd leave this out. Previously it was an assertion that could be verified, but now it no longer makes sense.